json_schemas/brief-responses-digital-outcomes-and-specialists-digital-outcomes.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "essentialRequirements": {
+      "items": {
+        "type": "boolean"
+      },
+      "maxItems": 10,
```

Comment:
We should check this number with Ralph and/or Sarah - I noticed this morning that one of the examples in the prototype has 11 essential requirements.  We can add a `number_of_items` key in the content to change the max items that can be added to the list.

But I guess deciding the number isn't essential for this PR.